src/test/java/com/wikia/webdriver/common/driverprovider/NewDriverProvider.java

```diff
@@ -102,22 +103,19 @@ public static WebDriver getWebDriver() {
   private static EventFiringWebDriver getIEInstance() {
     File file =
         new File("." + File.separator + "src" + File.separator + "test" + File.separator
-                 + "resources" + File.separator + "IEDriver" + File.separator
-                 + "IEDriverServer.exe");
+            + "resources" + File.separator + "IEDriver" + File.separator + "IEDriverServer.exe");
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "resources" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)